src/garage/torch/policies/continuous_nn_policy.py

```diff
@@ -0,0 +1,70 @@
+"""
+This modules creates a continuous policy network.
+
+A continuous neural network can be used as policy method in different RL
+algorithms. It accepts an observation of the environment and predicts an
+action.
+"""
+import akro
+import torch
+
+from garage.tf.policies.base import Policy
+
+
+class ContinuousNNPolicy(Policy):
+    """
+    Implements a module-agnostic policy network.
+
+    The policy network selects action based on the state of the environment.
+    It uses a PyTorch neural network module to fit the function of pi(s).
+    """
+
+    def __init__(self,
+                 env_spec,
+                 nn_module,
+                 policy_type='Deterministic',
```

Comment:
is this really cleaner than inheritance? @krzentner

Reply:
alternatively, we can make all policies stochastic and use a deterministic (dirac delta) distribution to implement determinism, but i'm not sure if that would mess up things like log_likelihood (probably would)

Reply:
Yeah, I was always intending us to use a "deterministic" distribution to implement this, and so that the API is always the same. I'm not sure where this `policy_type` idea came from, and I don't really like it, since it will makes API types "change at a distance."

Reply:
Can you please just delete `policy_type` and put it back to how it was? (i.e. forward returning a `Uniform` distribution covering only one value.)